docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -67,17 +60,8 @@ to restrict their use to the owner.
 * xref:Ownable-OwnershipTransferred[`++OwnershipTransferred(previous_owner, new_owner)++`]
 --
 
-[#Ownable-Utilities]
-==== Utilities
-
-[.contract-item]
-[[Ownable-assert_only_owner]]
-==== `[.contract-item-name]#++assert_only_owner++#++(self: @ContractState)++` [.item-kind]#internal#
-
-Panics if called by any account other than the owner.
-
-[#Ownable-Functions]
-==== Functions
+[#Ownable-External-Functions]
+==== External Functions
```

Comment:
what about putting `transfer_ownership` above? either way we should normalize across the docs to follow the code's definition order as much as possible